### ISSUE TITLE
Option to turn off the HP Bar and an option for Horizontal Bars?

### ISSUE THREAD

**Issue by Terroosh** [SheepGoMeh/SideHUD#1]
Would it be possible to get these two options?

**Comment by ryankhart** [comment]
Agreed. The reason why I want this as a healer WHM main is that I already have my eyes on the party list hud for so much of the time anyway for people's health bars, and I watch my cast bar all the time to be able to slide cast out of AOEs. So if I can just have a SideHUD MP bar with no HP bar, I can keep a better eye on my MP bar because SideHUD's MP bar and cast bar are one in the same.
